presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -688,7 +688,7 @@ public void run(ManagedQueryExecution query)
                 query.fail(new QueryQueueFullException(id));
                 return;
             }
-            query.setResourceGroupQueryLimits(group.getPerQueryLimits());
+            query.setResourceGroupQueryLimits(this.getPerQueryLimits());
```

Comment:
move this to the previous commit

Reply:
needs to be on second commit because enforcement of limits happens in second, so `ManagedQueryExecution::setPerQueryLimits` does not exist in first commit